src/components/TableOfContents/TableOfContentsMobile.tsx

```diff
@@ -27,46 +28,49 @@ const Mobile: React.FC<IPropsTableOfContentsMobile> = ({ items, maxDepth }) => {
   }
 
   return (
-    <Box
-      as="aside"
-      background="background"
-      border="1px"
-      borderColor="border"
-      borderRadius="4px"
-      py={2}
-      px={4}
-    >
-      <Flex
-        color="text200"
-        cursor="pointer"
-        alignItems="center"
-        justify="space-between"
-        {...getButtonProps()}
+    <Show below="l">
```

Comment:
This was the missing component to ensure that this component was only visible below the `l` value breakpoint, per pre-migration styling.